runtimevar/blobvar/blobvar.go

```diff
@@ -245,6 +275,10 @@ func (w *watcher) WatchVariable(ctx context.Context, prev driver.State) (driver.
 
 // Close implements driver.Close.
 func (w *watcher) Close() error {
+	if w.opener != nil {
+		w.opener.decBucketRef(w.bucket)
+		w.opener = nil // Ensure that we don't call multiple times.
```

Comment:
Should it also close the bucket when refs is 0?

Reply:
Yes. Done.